README.md

```diff
@@ -350,12 +350,34 @@ When creating Redshift tables, `spark-redshift`'s default behavior is to create
 
 To support larger columns, you can use the `maxlength` column metadata field to specify the maximum length of individual string columns. This can also be done as a space-savings performance optimization in order to declare columns with a smaller maximum length than the default.
 
-Here is an example of updating a column's metadata field in Scala:
+Here is an example of updating multiple columns' metadata fields:
 
-```
+```scala
 import org.apache.spark.sql.types.MetadataBuilder
-val metadata = new MetadataBuilder().putLong("maxlength", 10).build()
-df.withColumn("colName", col("colName").as("colName", metadata)
+
+// Specify the custom width of each column
+val columnLengthMap = Map(
+  "language_code" -> 2,
+  "country_code" -> 2,
+  "url" -> 2083
+)
+
+var df = ... // the dataframe you'll want to write to Redshift
+
+// Apply each column metadata customization
+columnLengthMap.foreach(
+  case (colName, length) => {
```

Comment:
You can still remove one level of braces, but I'll do this myself right now on merge.